### PR TITLE
 #162039621 Fix file download issue on travel-readiness

### PR DESCRIPTION
**Pull request by emekafredy**
#### What does this PR do?
Modify downloaded data display format for travel-readiness.

#### Description of Task to be completed?
* Modify column description titles
* Format returned dates
* make limit and page queries optional to allow for the download of all records

#### How should this be manually tested?
* Clone the repo with ``` git clone https://github.com/andela/travel_tool_back.git ```
* cd to travel_tool_back
* Checkout to the branch ``` bg-modify-travel-readiness-csv-data-field-titles-162039621 ```
* Run ``` yarn ``` to install dependencies
* Start up the server with ``` yarn start ```
* Open postman and add a valid token
* Make a get request with the endpoint ``` http://localhost:5000/api/v1/analytics/readiness?page=1&limit=7&type=file ``` or ``` http://localhost:5000/api/v1/analytics/readiness?type=file ``` (different query values can be applied).

#### Any background context you want to provide?
* You need to have the role of a ``` Travel Administrator ``` to access this route

#### What are the relevant pivotal tracker stories?
[162039621](https://www.pivotaltracker.com/story/show/162039621)

#### Screenshots (if appropriate)
screen shot 2018-11-20 at 11 34 00 pm

### CODE REVIEW

**Comment by Spectrumsun** [comment]
Update your PR label to Ready @emekafredy

**Comment by Spectrumsun** [comment]
fix your label @emekafredy

File: src/helpers/validators.js [hunk 1]
```diff
@@ -100,10 +100,6 @@ export const deleteChecklistItem = [
     .trim().isLength({ min: 3, }),
 ];
 export const travelReadinessValidators = [
```
[the remark this answers is not in the document]
Comment (Spectrumsun) [reply src/helpers/validators.js]: okay but the problem now is without validation it displays an error if the queries params are not correct in postman
image
@emekafredy

[the remark this answers is not in the document]
Comment (emekafredy) [reply src/helpers/validators.js]: Okay. I understand the side effect now. I will work on this right away. It should check for the format only if it is actually provided @Spectrumsun

**Comment by Spectrumsun** [comment]
LGTM

**Pull request merged**